modules/tinymce/src/core/main/ts/Deprecations.ts

```diff
@@ -19,7 +19,6 @@ const deprecatedSettings = (
   'paste_word_valid_elements,paste_retain_style_properties,paste_convert_word_fake_lists'
 ).split(',');
 
-const deprecatedPlugins = 'bbcode,colorpicker,contextmenu,fullpage,legacyoutput,spellchecker,textcolor'.split(',');
```

Comment:
This shouldn't have been touched yet as we may need to keep it around for 6.0 so people are aware when they upgrade and then remove it in 6.1 or later. It's not entirely clear what the plan is there yet, so we'll need to discuss it at some stage.

- [ ] Revert changes to this file

Reply:
What happens if a removed plugin is specified with this in place? Does keeping it really help?

Reply:
It'll print the console warning that it's been deprecated or removed so that integrators are aware they should remove it (we'll likely need to tweak the warning message if we decide to keep this).

Reply:
Reverted, we'll handle this message in a later task